.github/workflows/go-tests.yml

```diff
@@ -9,25 +9,32 @@ on:
 
 jobs:
   tests:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+        go-version:
```

Comment:
Could you add Go 1.11? since it's now a minimum supported Go version.

Reply:
Good call! Looks like 1.11 didn't work & it needed to be 1.13. Thanks 👍